docs/scenes/get-started/set-up-your-environment/instructions/androidPhysicalDevelopmentBuild.mdx

```diff
@@ -0,0 +1,54 @@
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+
+import { BuildEnvironmentSwitch } from '../BuildEnvironmentSwitch';
+
+## Set up an Android device with a development build
+
+<BuildEnvironmentSwitch type="android-studio" />
+
+<Step label="1">
+
+### Install EAS CLI
+
+To build your app, you will need to install EAS CLI. You can do this by running the following command in your terminal:
+
+<Terminal cmd={['$ npm install -g eas-cli']} />
+
+</Step>
+
+<Step label="2">
+
+### Create an Expo account and login
+
+To build your app, you will need to create an Expo account and login to the EAS CLI.
+
+1. [Sign up](https://expo.dev/signup) for an Expo account.
+2. Run the following command in your terminal to login to the EAS CLI:
```

Comment:
```suggestion
2. Run the following command in your terminal to log in to the EAS CLI:
```

- Let's prefer to use the log in as a verb here since this an action